src/components/TeacherComponents/LessonOverviewMediaClips/LessonOverviewMediaClips.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from "react";
+import { OakGrid, OakGridArea } from "@oaknational/oak-components";
+
+import LessonOverviewClipWithThumbnail from "./LessonOverviewClipWithThumbnail";
+
+import { MediaClipsList } from "@/node-lib/curriculum-api-2023/queries/lessonMediaClips/lessonMediaClips.schema";
+import { resolveOakHref } from "@/common-lib/urls";
+
+type LessonOverviewMediaClipsProps = {
+  learningCycleVideos: MediaClipsList;
+  unitSlug: string | null;
+  programmeSlug: string | null;
+  lessonSlug: string;
+};
+
+const LessonOverviewMediaClips: FC<LessonOverviewMediaClipsProps> = ({
+  learningCycleVideos,
+  unitSlug,
+  programmeSlug,
+  lessonSlug,
+}) => {
+  const videosArray = Object.values(learningCycleVideos);
+  return (
+    <OakGrid
+      $width={"100%"}
+      $gridTemplateColumns={["repeat(1, 1fr)", "repeat(4, 1fr)"]}
```

Comment:
I've created a bug ticket to break these cards to the same as mobile view when it gets to tablet:
https://www.notion.so/oaknationalacademy/I-want-the-OakMediaClipStackListItem-to-be-the-same-on-mobile-and-on-tablet-15126cc4e1b18009b8aff46cfa103965